packages/analytics-js-integrations/src/integrations/VWO/browser.js

```diff
@@ -26,11 +32,10 @@ class VWO {
       propagateEventsUntransformedOnError: this.propagateEventsUntransformedOnError,
       destinationId: this.destinationId,
     } = destinationInfo ?? {});
-    logger.debug('Config ', config);
+    logger.debug('Config', config);
```

Comment:
We're plainly dumping config data onto the console. Let's avoid doing this across all the integrations.

Reply:
removed debug statement